src/services/parser/message/__tests__/update-extract-for-sending.test.js

```diff
@@ -505,31 +514,64 @@ describe('updateExtractForSending', () => {
     const oldReceivingAsid = '200000001161';
     const oldEhrRequestId = 'BBBBA01A-A9D1-A411-F824-9F7A00A33757';
     const originalEhrExtract = templateEhrExtract(
-      oldReceivingAsid,
-      oldSendingAsid,
-      oldEhrRequestId,
-      'some-old-author-ods-code'
+        oldReceivingAsid,
+        oldSendingAsid,
+        oldEhrRequestId,
+        'some-old-author-ods-code'
     );
 
     const newSendingAsid = '200000001161';
     const newReceivingAsid = '200000001162';
     const ehrRequestId = v4();
     const expectedEhrExtract = templateEhrExtract(
-      newReceivingAsid,
-      newSendingAsid,
-      ehrRequestId,
-      'sending-ods-code'
+        newReceivingAsid,
+        newSendingAsid,
+        ehrRequestId,
+        'sending-ods-code'
     );
 
     const newEhrExtract = await updateExtractForSending(
-      originalEhrExtract,
-      ehrRequestId,
-      newReceivingAsid,
-      'sending-ods-code'
+        originalEhrExtract,
+        ehrRequestId,
+        newReceivingAsid,
+        'sending-ods-code'
     );
 
     const parsedNewEhrExtract = await new XmlParser().parse(newEhrExtract);
     const parsedExpectedEhrExtract = await new XmlParser().parse(expectedEhrExtract);
     expect(parsedNewEhrExtract).toEqual(parsedExpectedEhrExtract);
   });
+
+  it('should keep escaped special characters (linebreak, apostrophes, quotation marks) unchanged', async () => {
+    // given
+    const oldSendingAsid = '200000000149';
+    const oldReceivingAsid = '200000001161';
+    const oldEhrRequestId = 'BBBBA01A-A9D1-A411-F824-9F7A00A33757';
+    const originalEhrExtract = templateEhrExtract(
+        oldReceivingAsid,
+        oldSendingAsid,
+        oldEhrRequestId,
+        'some-old-author-ods-code'
+    );
+
+    // manually add some escaped special characters to the ehr extract
+    const replacedText = "Drinking status on eventdate: &#10;&#10;&#10; &apos; &quot; &Current drinker"
+    const ehrExtractWithSpecialChars = originalEhrExtract.replace(
+        "Drinking status on eventdate: Current drinker",
+        replacedText
+    )
```

Comment:
Can we add a semicolon ; here please?

```suggestion
    const ehrExtractWithSpecialChars = originalEhrExtract.replace(
        "Drinking status on eventdate: Current drinker",
        replacedText
    );
```